modules/launcher/src/main/java/org/mule/runtime/module/launcher/MuleContainer.java

```diff
@@ -217,6 +237,51 @@ protected void init(String[] args) throws IllegalArgumentException, Initialisati
     artifactResourcesRegistry.getMemoryManagementService().initialise();
     artifactResourcesRegistry.inject(artifactResourcesRegistry.getContainerProfilingService());
     initialiseIfNeeded(artifactResourcesRegistry.getContainerProfilingService());
+
+    if (isFipsEnabled()) {
+      configureSecurityManager();
+      setSecurityAlgorithm();
+    }
+  }
+
+  private void configureSecurityManager() throws InitialisationException {
```

Comment:
cant this method be something like configureSecurityManagerWithFipsPriority or something more specific if it is only used for fips?